src/utils/locale.ts

```diff
@@ -9,6 +9,14 @@ async function fetchLocaleStringsForComponent(componentName: string, locale: str
     });
   }
   
+  export function getLocaleFromDom(){
```

Comment:
Dette funker jo utmerket når jeg endrer på "lang"-attributtet på html-elementet.
Men siden du har et eget attributt for språk i varsom-observation-elementet, hvorfor bruker du ikke dette i stedet?
Jeg synes det var litt forvirrende at det er språk-attributtet i <varsom-observation> som styrer språket, mens formateringa styres av et annet attributt.

Reply:
Jørgen mente at språk burde styres av lang-attributtet i DOM og at vi ikke trengte language som prop. Nå er det lagt opp til at man kan overstyre språket ved å sette prop. Vet ikke hvor aktuelt det egentlig er, så kanskje vi skal fjerne language prop? Jeg skal prøve å endre metoden som sjekker DOM slik du beskriver.

Reply:
Enig. God ide å fjerne "language"-property. Bare husk å dokumentere hvordan man da velger språk.